### ISSUE TITLE
Keyboard user is unable to get the tooltips which are shown using mouse hover under "Monaco editor" but not appearing using tab navigation.

### ISSUE THREAD

**Issue by sagaranandani17** [microsoft/monaco-editor#1667]
**OS version:** 1903 (OS Build: 18326.295)
**Browser:** Edge(Microsoft Edge 44.18362.267.0)

**User Experience:** 
If tooltips are not accessible with keyboard than keyboard user will not be able to understand the control purpose and will face difficulties while interacting with it.

**Steps reproducing the issue:**
We are showing Css in Monaco editor and on hover tooltip is shown but not appearing using keyboard.

**Actual Result:**
Keyboard user is unable to get the tooltips which are shown using mouse hover under "Monaco editor" but not appearing using tab navigation.

**Expected Result:**
Keyboard user should be able to see the corresponding tooltip information when he/she navigates on to the Monaco editor control.

**Please find the attached video for more details**
[MAS2.1.1_Code editor (1).zip](https://github.com/microsoft/monaco-editor/files/3823203/MAS2.1.1_Code.editor.1.zip)

**Comment by alexdima** [comment]
The hover can be shown via the keyboard when the focus is in the editor on a symbol and when pressing `ctrl+k ctrl+i` or when pressing `F1 > Show Hover`
